en/metrics.rst

```diff
@@ -150,6 +149,19 @@ Geospatial Libraries
   <script type="text/javascript" src="https://www.openhub.net/p/liblas/widgets/project_factoids_stats?format=js"></script>
   <script type="text/javascript" src="https://www.openhub.net/p/python-iris/widgets/project_factoids_stats?format=js"></script>
 
+Coordinate Reference System Libraries 
```

Comment:
@kalxas, it looks like the tweak I put into metrics docs got pulled in here.
Agree, we should now only reference proj4 from the MetaCRS list now. There are a number of projects which should be getting removed from metrics.rst now that we have decided on the final list for version 11.